recipes/r-gdalraster/meta.yaml

```diff
@@ -0,0 +1,134 @@
+# License: MIT + file LICENSE
+# Copyright: See file inst/COPYRIGHTS for details.
+# URL: https://usdaforestservice.github.io/gdalraster/, https://github.com/USDAForestService/gdalraster
+# BugReports: https://github.com/USDAForestService/gdalraster/issues
+# Depends: R (>= 4.2.0)
+# Imports: bit64, graphics, grDevices, methods, nanoarrow, Rcpp (>= 1.0.7), stats, tools, utils, wk, xml2
+# LinkingTo: nanoarrow, Rcpp, RcppInt64
+# Suggests: gt, knitr, rmarkdown, scales, testthat (>= 3.0.0)
+# NeedsCompilation: yes
+# SystemRequirements: C++17, GDAL (>= 3.1.0, built against GEOS)
+# Encoding: UTF-8
+# RoxygenNote: 7.3.2
+# VignetteBuilder: knitr
+# Config/testthat/edition: 3
+# Packaged: 2025-07-11 23:34:52 UTC; ctoney
+# Author: Chris Toney [aut, cre] (R interface/additional functionality), Michael D. Sumner [ctb], Frank Warmerdam [ctb, cph] (GDAL API documentation; src/progress_r.cpp from gdal/port/cpl_progress.cpp), Even Rouault [ctb, cph] (GDAL API documentation), Marius Appel [ctb, cph] (configure.ac based on https://github.com/appelmar/gdalcubes), Daniel James [ctb, cph] (Boost combine hashes method in src/cmb_table.h), Peter Dimov [ctb, cph] (Boost combine hashes method in src/cmb_table.h)
+# Maintainer: Chris Toney <jctoney@gmail.com>
+# Repository: CRAN
+# Date/Publication: 2025-07-14 19:30:03 UTC
```

Comment:
```suggestion
```

This is all cruft that shouldn't exist, but hasn't gotten removed for some reason from the recipe generator.